content/hi/docs/tasks/tools/included/verify-kubectl.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें."
+headless: true
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) की आवश्यकता होती है,
+जो स्वचालित रूप से तब बनता है जब आप
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डेप्लॉय करते हैं.
```

Comment:
```suggestion
या मिनीक्यूब क्लस्टर को सफलतापूर्वक डिप्लॉय करते हैं.
```

Reply:
i would suggest always refer to the sheet here: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0
As many words have been added to that!!